src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -120,7 +131,7 @@ private void PrepareRequestCookies(HttpRequest request, HttpWebRequest webReques
                     {
                         persistentCookieContainer.Add(new Cookie(pair.Key, pair.Value, "/", request.Url.Host)
                         {
-                            Expires = DateTime.UtcNow.AddHours(1)
+                            Expires = DateTime.Now.AddHours(1)
```

Comment:
I'm in England (so UTC+1 at the moment) and when I run the tests on linux, a cookie with expiry UtcNow.AddHours(1) shows as expired as soon as it's created so it never gets sent.  I've put a test case here showing the problem:
https://gist.github.com/ta264/7822b1424f72e5b4c961

If you prefer, switching to DateTime.UtcNow.AddHours(24) should also work around the problem.